edi_storage_oca/models/edi_exchange_record.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class EDIExchangeRecord(models.Model):
+    _inherit = "edi.exchange.record"
+
+    storage_id = fields.Many2one(
```

Comment:
Sorry, I missed on step: for consistency this field should be automatically populated for existing records via a proper sql query to update them all at once on post-migrate.